test/feature/step_definitions/setup_definitions.coffee

```diff
@@ -1,30 +1,27 @@
+async = require 'async'
 { expect } = require 'chai'
 
 
 module.exports = ->
 
   @Given /^an exprestive app using defaults$/, (done) ->
-    @createExprestiveApp '', (err) =>
-      return done.fail err if err
-      @startApp (err) ->
-        return done.fail err if err
-        done()
+    async.series [
+      (next) => @createExprestiveApp '', next
+      (next) => @startApp next
+    ], done
```

Comment:
With Nitroglycerin this would be even more concise and readable:

```coffeescript
@Given /^an exprestive app using defaults$/, (done) ->
  @createExprestiveApp '', N =>
    @startApp (err) done
```